bot/audiodownloader.py

```diff
@@ -0,0 +1,13 @@
+import pafy
+import os
+
+
+def audio_downloader(url): 
```

Comment:
Add docstring for this function

Reply:
Make [Type Hinting](url) for this function
```suggestion
def audio_downloader(url: str) -> str: 
```

Reply:
**Outdated**